open-api/rest-catalog-open-api.yaml

```diff
@@ -147,6 +147,7 @@ paths:
                   "defaults": {
                     "clients": "4"
                   },
+                  "idempotencyKeyLifetime": "PT30M",
```

Comment:
I feel like this is inconsistent with most of our property formatting which is primarily `kebab-case`.  This should probably be `idempotency-key-lifetime`.

Reply:
Fixed. Thanks